### PR TITLE
maint: updating `ansys/actions` to `v6`

### PR DESCRIPTION
**Pull request by clatapie**
As title says.

### CODE REVIEW

**Comment by clatapie** [comment]
Another PR will follow to modify the `dependabot.yml` file as there was no opened PR related to the `ansys/actions` releases.

File: .github/workflows/ci_cd.yml [hunk 1]
```diff
@@ -47,7 +47,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: "PyAnsys documentation style checks"
-        uses: ansys/actions/doc-style@v5
+        uses: ansys/actions/doc-style@v6
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (RobPasMue) [inline .github/workflows/ci_cd.yml:52]: You'll need to adapt vale most probably

**Review by RobPasMue — CHANGES_REQUESTED** [review]

File: .github/workflows/ci_cd.yml (hunk 1 above)
Comment (RobPasMue) [reply .github/workflows/ci_cd.yml:52]: You got a silent failure: https://github.com/ansys/pyansys-math/actions/runs/8738934103/job/23979064484?pr=314#step:2:178

**Review by RobPasMue — CHANGES_REQUESTED** [review]

File: .github/workflows/ci_cd.yml (hunk 1 above)
Comment (clatapie) [reply .github/workflows/ci_cd.yml:52]: True, I need to modify it. Thanks for catching the issue!

**Review by RobPasMue — APPROVED** [review]

**Pull request merged**